packages/splits-sdk-react/src/hooks/splitV1.ts

```diff
@@ -113,6 +113,12 @@ export const useCreateSplit = (): {
         setError(undefined)
         setTxHash(undefined)
 
+        const chainId = splitsClient._walletClient?.chain.id
```

Comment:
this can be a utility function, see this repeated in a few places
maybe 2 functions or just 1, something like `getChainIdFromWalletClient`